src/main/resources/application.properties

```diff
@@ -1 +1,6 @@
+app.name=Illuminati on Local
+app.description=${app.name}
+spring.profiles.active=${SPRING_ACTIVE_PROFILE:"dev"}
```

Comment:
spring.profiles.active=${SPRING_ACTIVE_PROFILE:""} to set it to default

Reply:
Updated